benchmarks/nHcal/nhcal_acceptance/config.yml

```diff
@@ -0,0 +1,40 @@
+sim:nhcal_acceptance:
+  extends: .det_benchmark
+  stage: simulate
+  parallel:
+    matrix:
+      - ENERGY: ["1GeV", "5GeV", "10GeV"]
+        INDEX: ["00","01","02","03","04","05","06","07","08","09"]
+  script:
+    - snakemake --cores 5 sim_output/nhcal_acceptance/E${ENERGY}/sim_epic_backward_hcal_only.${INDEX}.edm4hep.root
+
+bench:nhcal_acceptance_combine:
+  extends: .det_benchmark
+  stage: benchmarks
+  needs: 
+    - "sim:nhcal_acceptance"
+  parallel:
+    matrix:
+      - ENERGY: ["1GeV", "5GeV", "10GeV"]
+  script:
+    - snakemake --cores 5 sim_output/nhcal_acceptance/sim_epic_backward_hcal_only_E${ENERGY}_combined_10files.edm4hep.root
+
+
+bench:nhcal_acceptance_analysis:
+  extends: .det_benchmark
+  stage: benchmarks
+  needs:
+    - "bench:nhcal_acceptance_combine"
+  parallel:
+    matrix:
+      - ENERGY: ["1GeV", "5GeV", "10GeV"]
+  script:
+    - snakemake --cores 3 results/nhcal_acceptance_analysis_epic_backward_hcal_only_E${ENERGY}_combined_10files.pdf
+
+collect_results:nhcal_acceptance:
+  extends: .det_benchmark
+  stage: collect
+  needs: 
+    - "bench:nhcal_acceptance_analysis"
+  script:
+    - ls -lrht
```

Comment:
You could add an extra step to convert plots to png like in
https://github.com/eic/detector_benchmarks/blob/2c0dd9b63972ca862d98b73aaa931571f49e1e1f/benchmarks/tracking_performances_dis/config.yml#L34-L38
only png files are currently rendered by jlab image browser.